### ISSUE TITLE
Where can I find the training dataset? 

### ISSUE THREAD

**Issue by zhewang95** [zeusees/Mobilenet-SSD-License-Plate-Detection#7]
or if the dataset will not be published, can you tell me how many images are used in training?

**Comment by soufianesabiri** [comment]
I want to retrain the model for other types of license plates, but I don't find any way on how to do it...

**Comment by lsy17096535** [comment]
@zhewang95  about 8000 images

**Comment by lsy17096535** [comment]
@soufianesabiri  you can refer this blog https://blog.csdn.net/weixin_39750664/article/details/82502302

**Comment by soufianesabiri** [comment]
@lsy17096535 Thanks! 
I found this also: https://tolotra.com/2018/09/15/how-to-retrain-ssd-mobilenet-for-real-time-object-detection-using-a-raspberry-pi-and-movidius-neural-compute-stick/

Can I use it? Is it the same procedure?

Also, there is this file: https://github.com/tensorflow/hub/blob/master/examples/image_retraining/retrain.py

Which I found in another article, they used a command like this:

`python retrain?py --how_many_training_steps=500 --output_graph=out.pb --output_labels=labels.txt --image_dir=lp --tfhub_module https://tfhub.dev/google/imagenet/mobilenet_v1_100_224/feature_vector/1`

To generate the pb file mentionned in the HyperLPR baidu link: https://pan.baidu.com/s/11V0Pfx6jGNaBOOTqDSVK2Q

**Comment by lsy17096535** [comment]
@soufianesabiri  If you want to train a caffe model, you can refer this blog. Other framework you should read similar blog.

**Comment by soufianesabiri** [comment]
Ok I will, thank you for your answer!
